### PR TITLE
Secure comparison. 

### PR DESCRIPTION
**Pull request by Anu1601CS**
Pull Request for Issue #51

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_templates.ini [hunk 1]
```diff
@@ -230,7 +230,7 @@ COM_TEMPLATES_TEMPLATE_PREVIEW="Preview"
 COM_TEMPLATES_TEMPLATES_FILTER_SEARCH_DESC="Search in template name or folder name."
 COM_TEMPLATES_TOGGLE_FULL_SCREEN="Press Ctrl-Q to toggle Full Screen editing."
 COM_TEMPLATES_TOOLBAR_SET_HOME="Default"
-COM_TEMPLATES_UPDATED="%s Updated"
+COM_TEMPLATES_UPDATED="%s Conflict found."
```
[the remark this answers is not in the document]
Comment (Anu1601CS) [reply administrator/language/en-GB/en-GB.com_templates.ini]: @astridx Can we use `conflict(s)` ?

[the remark this answers is not in the document]
Comment (astridx) [reply administrator/language/en-GB/en-GB.com_templates.ini]: @anu1601cs
Why is it not ppssible to use plural?

@laneO
You suggested the change of thi's label text. What do you think?

[the remark this answers is not in the document]
Comment (Anu1601CS) [reply administrator/language/en-GB/en-GB.com_templates.ini]: > @Anu1601CS
Why is it not ppssible to use plural?

@astridx It was a suggestion. We can use as you have suggested.

[the remark this answers is not in the document]
Comment (laoneo) [reply administrator/language/en-GB/en-GB.com_templates.ini]: Agree with plural feature usage from core.

**Comment by Anu1601CS** [comment]
@laoneo @astridx Need review

**Pull request merged**